packages/core/contracts-ovm/insured-bridge/implementation/OVM_BridgeDepositBox.sol

```diff
@@ -48,8 +48,8 @@ contract OVM_BridgeDepositBox is OVM_CrossDomainEnabled, OVM_Testable {
      *  OVM DEPOSIT BOX DATA STRUCTURES  *
      *************************************/
 
-    // Address of the L1 bridge router that acts as the owner of this Bridge deposit box.
-    address public bridgeRouter;
+    // Address of the L1 contract that acts as the owner of this Bridge deposit box.
+    address public bridgeAdmin;
```

Comment:
@chrismaree I felt this name was a bit more accurate and better than `bridgePoolFactory`

Reply:
cool. do you want to rename the `BridgePoolFactory` to `BridgeAdmin` as well?

Reply:
I think `BridgePoolFactory` is more accurate since its a factory, but from the DepositBox's perspective its the admin. What do you think?

Reply:
I see you updated this. looks good.